aten/src/ATen/native/cuda/SoftMax.cu

```diff
@@ -12,13 +12,17 @@
 #include <ATen/cuda/NumericLimits.cuh>
 #include <type_traits>
 
+#include <ATen/native/cuda/Loops.cuh>
```

Comment:
How is this file used?

Reply:
This is needed for the following line `#include <ATen/native/cuda/MemoryAccess.cuh>` to compile successfully. `MemoryAccess.cuh` needs some values from `Loops.cuh`